pombola/core/models.py

```diff
@@ -821,6 +821,10 @@ def get_absolute_url(self):
     class Meta:
        ordering = ["name"]
 
+    @property
+    def is_election_list(self):
+        return self.kind.slug == 'election-list'
+
```

Comment:
I don't think we need to put this in a method - while we generally want small functions that do one thing, this is a function that's specific to a data value, so we'd end up with a whole bunch of these eventually.